wgpu-core/src/command/render.rs

```diff
@@ -1473,10 +1478,31 @@ impl Global {
                         )));
                     }
 
+                    // If this.depthLoadOp is "clear", this.depthClearValue must be provided and must be between 0.0 and 1.0, inclusive.
+                    if depth_stencil_attachment.depth.load_op == Some(LoadOp::Clear) {
+                        if let Some(clear_value) = depth_stencil_attachment.depth.clear_value {
+                            if !(0.0..=1.0).contains(&clear_value) {
+                                return Err(CommandEncoderError::InvalidAttachment(AttachmentError::ClearValueOutOfRange));
+                            }
+                        } else {
+                            return Err(CommandEncoderError::InvalidAttachment(AttachmentError::NoClearValue));
+                        }
+                    }
+
+                    let depth = {
+                        let PassChannel { load_op, store_op, clear_value, read_only } = depth_stencil_attachment.depth;
```

Comment:
Might be worth a note why struct update syntax won't work here (because of different generic)